uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveBalanceChangeTest.php

```diff
@@ -119,49 +119,65 @@ public function testBalanceForEntitlementCanSumOnlyTheBalanceChangesForLeaveRequ
     $this->createLeaveBalanceChange($entitlement->id, 10);
     $this->assertEquals(10, LeaveBalanceChange::getBalanceForEntitlement($entitlement));
 
-    $this->createLeaveRequestBalanceChange(
-      $entitlement->type_id,
-      $entitlement->contact_id,
-      $leaveRequestStatuses['Cancelled'],
-      date('Y-m-d', strtotime('-10 days'))
-    );
+    LeaveRequestFabricator::fabricateWithoutValidation([
```

Comment:
The move from helper methods to Fabricator was a good idea! Please keep it when adding a new test to check that the method does not include deleted records